Caddyfile

```diff
@@ -10,7 +10,4 @@ YOUR_FQDN_OR_HOSTNAME {
 	log {
 		output file /var/log/caddy.log
 	}
-	handle_path /api/* {
```

Comment:
I'd rather we don't just remove it. The API is useful! I'd propose we move this `Caddyfile` to `Caddyfile.sample` with the api intact, and then add `Caddyfile` to `.gitignore` and then we'll add this as a step for users during setup.

As for the reason the API was removed here to begin with, let's do a PR tonight (me or you) that cleanses the config output.